doc/docs.md

```diff
@@ -1993,20 +1993,17 @@ mut p := Point{
 	x: 10
 	y: 20
 }
-p = Point{
-	x: 30
-	y: 4
-}
+
+// The field name can be omitted.
+p = Point{30, 4}
```

Comment:
In my opinion, that is *the only change* that is valuable here. The rest should not be part of the PR - they worsened the quality of the documentation by commenting working code, and accentuating very minute details :-| .

Reply:
Huh?my bad, I thought it was commented out from the start, but I must have been mistaken.

Reply:
So, `Omitting the struct name also works for returning a struct literal or passing one
as a function argument.` also should be restored?

Reply:
This is still a good improvement as the quality of this section is very poor to begin with.
The last two lines are now unnecessary.

Reply:
As I already said, I also think that these 2 lines (1997 and 1998) are good - they are the only thing that should be in the PR.